Upstream-Status: Backport

Slightly changed from the upstream version, which referenced
base/memory/shared_memory_tracker.cc that does not exist in 58.0.3029.81.

Signed-off-by: Raphael Kubo da Costa <raphael.kubo.da.costa@intel.com>
---
From 379e35f6f3eaa41a97f2659249509ca599749b27 Mon Sep 17 00:00:00 2001
From: rayb <rayb@ca.ibm.com>
Date: Fri, 10 Mar 2017 16:25:29 -0800
Subject: [PATCH] Fix GN bootstrap.

Added missing files.

BUG=none
R=jochen@chromium.org, thakis@chromium.org

Review-Url: https://codereview.chromium.org/2747483002
Cr-Commit-Position: refs/heads/master@{#456229}
---
 tools/gn/bootstrap/bootstrap.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tools/gn/bootstrap/bootstrap.py b/tools/gn/bootstrap/bootstrap.py
index 38cfb11..75e2eb4 100755
--- a/tools/gn/bootstrap/bootstrap.py
+++ b/tools/gn/bootstrap/bootstrap.py
@@ -384,6 +384,7 @@ def write_gn_ninja(path, root_gen_dir, options):
       'base/base_paths.cc',
       'base/base_switches.cc',
       'base/build_time.cc',
+      'base/callback_helpers.cc',
       'base/callback_internal.cc',
       'base/command_line.cc',
       'base/debug/activity_tracker.cc',
-- 
2.9.3

